src/parsers/perf/perfparser.cpp

```diff
@@ -811,8 +811,17 @@ struct PerfParserPrivate
         auto args = features.cmdline;
         args.removeFirst();
         summaryResult.command = QLatin1String("perf ") + QString::fromUtf8(args.join(' '));
-
-        // TODO: add system info to summary page
+        summaryResult.hostName = QString::fromUtf8(features.hostName);
+        summaryResult.linuxKernelVersion = QString::fromUtf8(features.osRelease);
+        summaryResult.perfVersion = QString::fromUtf8(features.version);
+        summaryResult.cpuDescription = QString::fromUtf8(features.cpuDesc);
+        summaryResult.cpuId = QString::fromUtf8(features.cpuId);
+        summaryResult.cpuArchitecture = QString::fromUtf8(features.arch);
+        summaryResult.cpusOnline = features.nrCpusOnline;
+        summaryResult.cpusAvailable = features.nrCpusAvailable;
+        summaryResult.cpuSiblingCores = QString::fromUtf8(features.siblingCores.join(", ")).prepend(QLatin1Char('[')).replace(QLatin1String(", "), QLatin1String("], [")).append(QLatin1Char(']'));
```

Comment:
this and the following line needs to be reformatted to not be so long. I think something like this would be good, and it would also simplify the code a bit (note how you insert ", " and then replace it again later). Or am I missing something?

        auto formatCpuList = [](const QByteArrayList& list) -> QString {
            return QString::fromUtf8('[' + list.join("], [") + ']');
        };
        summaryResult.cpuSiblingCores = formatCpuList(features.siblingCores);
        summaryResult.cpuSiblingThreads = formatCpuList(features.siblingThreads);